api-controllers/ServiceFabrikApiController.js

```diff
@@ -326,15 +348,17 @@ class ServiceFabrikApiController extends FabrikBaseController {
   getLastBackup(req, res) {
     req.manager.verifyFeatureSupport('backup');
     return eventmesh.apiServerClient.getLastOperation({
-        resourceId: req.params.instance_id,
+        resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT,
+        resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
         operationName: CONST.OPERATION_TYPE.BACKUP,
-        operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP
+        operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
+        resourceId: req.params.instance_id
       })
       .then(backupGuid =>
-        eventmesh.apiServerClient.getOperationResponse({
-          operationName: CONST.OPERATION_TYPE.BACKUP,
-          operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
-          operationId: backupGuid,
+        eventmesh.apiServerClient.getResponse({
+          resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.BACKUP,
+          resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
+          resourceId: backupGuid
```

Comment:
renamed it to `getLastOperationValue`

Reply:
I believe you are talking about [`status.lastOperation`](https://github.com/muralisc/service-fabrik-broker/wiki/Resource-structure,-Locking-and-Unlocking-with-apiserver) which is not being used by backup operation anywhere.
Although for deployment resources we are storing lastOperation object (What you mentioned) in there. And CF lastOperation call can directly get lastOperation from `status.lastOperation`
`status.lastOperation` is not used for backup probably because we don't want cf to know about backup and pollute lastOperation field. Maybe @subhankarc can pitch in here

Reply:
As discussed, we should not label deployment resource with `last_backup_defaultbackups` rather query backup resources with instance_guid